packages/terra-form-input/tests/wdio/form-input-spec.js

```diff
@@ -161,4 +161,16 @@ Terra.describeViewports('Form-Input', ['tiny', 'large'], () => {
       Terra.it.validatesElement();
     });
   });
+
+  describe('Setting Value', () => {
+    before(() => browser.url('/#/raw/tests/terra-form-input/form-input/setting-value-input'));
+
+    Terra.it.validatesElement('empty');
+
+    it('should not trigger onInput when programmatically setting value', () => {
+      browser.click('#input-set-text-button');
+    });
+
+    Terra.it.validatesElement('programmatically');
```

Comment:
I don’t think we need to test this code changes in multiple viewports. since the code changes were more of functional changes and has no changes in UI.

Reply:
Updated here https://github.com/cerner/terra-core/pull/3135/commits/d86c75de66b8308985db499365a1c8ae731331f9